src/process/process.lisp

```diff
@@ -157,62 +157,59 @@ NOTES:
   + `PROCESS-PERUSE-INBOX?' is passed along to `RECEIVE-MESSAGE', where it determines how we search for a message to handle.
```

Comment:
NTS: didn't look at the docstrings in this file, would be good to double-check whether they need tweaks